src/firmware/main/app/control/bangbang.h

```diff
@@ -64,38 +62,32 @@ typedef struct
 
 
 // wrappers to fill out the Profile struct with the input information
-void PrepareBBTrajectoryMaxV(BBProfile *b, float d, float vi, float vf, float MaxA,
-                             float MaxV);
-void PrepareBBTrajectory(BBProfile *b, float d, float vi, float vf, float MaxA);
+void app_bangbang_prepareTrajectoryMaxV(BBProfile *b, float d, float vi, float vf,
+                                        float MaxA, float MaxV);
+void app_bangbang_prepareTrajectory(BBProfile *b, float d, float vi, float vf,
+                                    float MaxA);
 
 // meat and potatoes, computes the relavent details of the trajectory
-void PlanBBTrajectory(BBProfile *b);
+void app_bangbang_planTrajectory(BBProfile *b);
 
 // the default acceleration calculator, a typical horizon would be on
 // the time scale fo the vehicle in question say 1/2 second for our robots.
```

Comment:
```suggestion
// the time scale of the vehicle in question say 1/2 second for our robots.
```
should the 1/2 second actually be 0.05 seconds as mentioned in physbot

Reply:
Changed as suggested and removed the bit with specific values